src/Translator/src/TranslationsDumper.php

```diff
@@ -166,7 +166,7 @@ private function generateConstantName(string $translationId): string
 
         $prefix = 0;
         do {
-            $constantName = s($translationId)->ascii()->snake()->upper()->toString().($prefix > 0 ? '_'.$prefix : '');
+            $constantName = preg_replace('/^(\d)/', '_$1', s($translationId)->ascii()->snake()->upper()->toString().($prefix > 0 ? '_'.$prefix : ''));
```

Comment:
LGTM, but I would be in favor to use `->replaceMatches` from `s()` instead of calling `preg_replace` directly

Reply:
Thanks for the review! Yeah I agree - I've changed it.